index.js

```diff
@@ -68,28 +89,13 @@ class ExecutorRouter extends Executor {
      * @return {Promise}
      */
     _start(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
+        const options = this.getFullConfig(config);
 
-        return executor.start(config);
-    }
-
-    /**
-     * Stop a running or finished build
-     * @method _stop
-     * @param {Object} config               Configuration
-     * @param {Object} [config.annotations] Optional key/value object
-     * @param {String} config.buildId       Unique ID for a build
-     * @return {Promise}
-     */
-    _stop(config) {
-        const annotations = config.annotations || {};
-        const executorType = annotations[ANNOTATION_EXECUTOR_TYPE];
-        const executor = this[executorType] || this.defaultExecutor; // Route to executor (based on annotations) or use default executor
+        // TODO: mark job as queued
 
-        return executor.stop(config);
+        // Note: arguments to enqueue are [queue name, job type, array of args]
+        return this.queue.enqueue('queuedJobs', 'start', [options]);
```

Comment:
I hadn't actually thought of that, that's a really good idea. I think it'll work, will check that now.